### PR TITLE
Update bioimageio_model

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.6326366.
The following version(s) will be added:
  - 6334888

You can find their corresponding RDF files here:
  - https://zenodo.org/api/files/e6b1c734-176b-4f20-bd76-eb1ce559ef5a/rdf.yaml

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the BioImage.IO Spec validator (static validation);
- [ ] Passed the BioImage.IO CI tests (dynamic validations);
- [ ] The meta information for the RDF item is complete;
  - [ ] The tags is complete and describes the model;
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D);
  - [ ] Authors are provided;
  - [ ] Documentation is complete;
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: none specified

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: https://github.com/bioimage-io/collection-bioimage-io/pull/321, https://github.com/bioimage-io/collection-bioimage-io/pull/324, https://github.com/bioimage-io/collection-bioimage-io/pull/328, https://github.com/bioimage-io/collection-bioimage-io/pull/329

### CODE REVIEW

**Comment by constantinpape** [comment]
@FynnBe 
```
 Traceback (most recent call last):
  File "scripts/dynamic_validation.py", line 5, in <module>
    import typer
ModuleNotFoundError: No module named 'typer'
```
shouldn't this always be pulled in as a dependency of biomiageio.core???

**Comment by FynnBe** [comment]
> @FynnBe
> 
> ```
>  Traceback (most recent call last):
>   File "scripts/dynamic_validation.py", line 5, in <module>
>     import typer
> ModuleNotFoundError: No module named 'typer'
> ```
> 
> shouldn't this always be pulled in as a dependency of biomiageio.core???

yes, it is a core (and also spec) dependency... ?

**Comment by FynnBe** [comment]
```yaml
channels: [conda-forge]
dependencies:
- python>=3.7,<3.8
- pip
- pip: ['tensorflow>=1.15,<2.0', csbdeep>=0.6.3, scikit-image, numba]
name: '6334888'

```
... we are missing the bioimageio.core dependency... I suppose we should always inject it, no?

**Comment by FynnBe** [comment]
> ... we are missing the bioimageio.core dependency... I suppose we should always inject it, no?

or do we want model developers to specify it in their conda envs? (somewhat more explicit?)

**Comment by constantinpape** [comment]
> > 
> 
> or do we want model developers to specify it in their conda envs? (somewhat more explicit?)

No, we have to inject it. A model should be valid independent of whether or not it has bioimageio.core in the dependencies.
(E.g. Since some python tool might decide to use it without it; I wouldn't recommend it, but we shouldn't rule it out completely).

**Comment by constantinpape** [comment]
> No, we have to inject it. A model should be valid independent of whether or not it has bioimageio.core in the dependencies.
> (E.g. Since some python tool might decide to use it without it; I wouldn't recommend it, but we shouldn't rule it out completely).

Note: we might run into some corner cases where the dependencies are incompatible with `bioimageio.core`, but I would not worry about this for now.

**Comment by FynnBe** [comment]
> Note: we might run into some corner cases where the dependencies are incompatible with bioimageio.core, but I would not worry about this for now. 

yes, we have fairly light dependencies...

**Comment by constantinpape** [comment]
Hooray, it works.

**Comment by constantinpape** [comment]
@uschmidt83 the upload of the H&E model is here and passing tests now.

Should I go ahead and merge this? It would mean that I would need to update this model if anything else needs to change with it, but that would be fine by me. (I would also change the name; but if we go ahead with this model I would do it via bioimage.io to also document this functionality).

**Comment by uschmidt83** [comment]
> Should I go ahead and merge this? It would mean that I would need to update this model if anything else needs to change with it, but that would be fine by me. (I would also change the name; but if we go ahead with this model I would do it via bioimage.io to also document this functionality).

Hmm, I would like to be able to update this myself in the future, if necessary.
Is it possible to transfer (or share) ownership on Zenodo?

**Comment by uschmidt83** [comment]
Also, did't @oeway ask to first rename the model to something recognizable before merging?

**Comment by constantinpape** [comment]
> FYI: I have done the transfer with the help from the Zenodo team, you can send a ticket to zenodo support and specify which depoist to transfer and which is the target account. Then the receiving account needs to send a confirmation email. After that, the zenodo team will help transferring the deposit.

Thanks for the info, @oeway. But I think here it's easier to just block the current model and let @uschmidt83 upload the model again.

File: collection/10.5281/zenodo.6326366/resource.yaml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 status: accepted
```
Comment (constantinpape) [inline collection/10.5281/zenodo.6326366/resource.yaml:1]:
```suggestion
status: blocked
```

File: collection/10.5281/zenodo.6326366/resource.yaml [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
 versions:
+- version_id: '6334888'
+  doi: 10.5281/zenodo.6334888
+  created: '2022-03-07 15:23:36.540825'
+  status: accepted
```
Comment (constantinpape) [inline collection/10.5281/zenodo.6326366/resource.yaml:6]:
```suggestion
  status: blocked
```

**Pull request merged**